mod/gc_tags/actions/blog/save.php

```diff
@@ -1,193 +1,193 @@
-				$values[$name] = elgg_get_excerpt($value);
-			}
-			break;
-
-		case 'container_guid':
-			// this can't be empty or saving the base entity fails
-			if (!empty($value)) {
-				if (can_write_to_container($user->getGUID(), $value)) {
-					$values[$name] = $value;
-				} else {
-					$error = elgg_echo("blog:error:cannot_write_to_container");
-				}
-			} else {
-				unset($values[$name]);
-			}
-			break;
-
-		default:
-			$values[$name] = $value;
-			break;
-	}
-}
-
-//implode for tranlation
-$values['title'] = gc_implode_translation($values['title'], $values['title2']);
-$values['excerpt'] = gc_implode_translation($values['excerpt'], $values['excerpt2']);
-$values['description'] =gc_implode_translation($values['description'], $values['description2']);
-
-// if preview, force status to be draft
-if ($save == true) {
-	$values['status'] = 'draft';
-}
-
-// if draft, set access to private and cache the future access
-if ($values['status'] == 'draft') {
-	$values['future_access'] = $values['access_id'];
-	$values['access_id'] = ACCESS_PRIVATE;
-}
-
-// assign values to the entity, stopping on error.
-if (!$error) {
-	foreach ($values as $name => $value) {
-		if (($name != 'title2') && ($name != 'description2') &&  ($name != 'excerpt2')){ // remove input 2 in metastring table
-		$blog->$name = $value;
-		}
-	}
-}
-
-// only try to save base entity if no errors
-if (!$error) {
-	if ($blog->save()) {
-		// remove sticky form entries
-		elgg_clear_sticky_form('blog');
-
-		// remove autosave draft if exists
-		$blog->deleteAnnotations('blog_auto_save');
-
-		// no longer a brand new post.
-		$blog->deleteMetadata('new_post');
-
-		// if this was an edit, create a revision annotation
-		if (!$new_post && $revision_text) {
-			$blog->annotate('blog_revision', $revision_text);
-		}
-
-		system_message(elgg_echo('blog:message:saved'));
-		$status = $blog->status;
-
-		// add to river if changing status or published, regardless of new post
-		// because we remove it for drafts.
-		if (($new_post || $old_status == 'draft') && $status == 'published') {
-			elgg_create_river_item(array(
-				'view' => 'river/object/blog/create',
-				'action_type' => 'create',
-				'subject_guid' => $blog->owner_guid,
-				'object_guid' => $blog->getGUID(),
-			));
-
-			elgg_trigger_event('publish', 'object', $blog);
-
-			// reset the creation time for posts that move from draft to published
-			if ($guid) {
-				$blog->time_created = time();
-				$blog->save();
-			}
-		} elseif ($old_status == 'published' && $status == 'draft') {
-			elgg_delete_river(array(
-				'object_guid' => $blog->guid,
-				'action_type' => 'create',
-			));
-		}
-
-		if ($blog->status == 'published' || $save == true) {
-			forward($blog->getURL());
-		} else {
-			forward("blog/edit/$blog->guid");
-		}
-	} else {
-		register_error(elgg_echo('blog:error:cannot_save'));
-		forward($error_forward_url);
-	}
-} else {
-	register_error($error);
-	forward($error_forward_url);
-}
+<?php
```

Comment:
For some reason this file is getting marked changed. Did you fix bad line endings or something?

Reply:
i think i just deleted everything and then did an undo (i needed to figure out which blog/save it was using), it shouldn't pick up on that change though